### PR TITLE
Create a website for AdminNetworkPolicy

### PR DESCRIPTION
**Pull request by astoycos**
Working to create a website documenting the AdminNetworkPolicy API, modeled after the [Gateway API one](https://gateway-api.sigs.k8s.io/)

### CODE REVIEW

**Comment by astoycos** [comment]
The first draft of this is ready to go, looking into what perms I need to actually deploy this via gh pages.

File: site-src/guides/api-overview.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+were being incorrectly used to do so, creating a complex web of objects to be
+maintained.
+
+With the advent of the AdminNetworkPolicy API Cluster Admins will now have the 
+ability to apply policy on in-cluster workloads with only a few simple policy 
+objects that can be applied globally.
+
+## Roles and personas
+
+In this documentation we refer to three primary personas:
```
Comment (bpickard22) [inline site-src/guides/api-overview.md:14]: I would add a brief description to these roles, like a one liner

File: site-src/guides/api-overview.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+There are two main objects in the AdminNetworkPolicy API resource model
+
+- **AdminNetworkPolicy (ANP)**
+
+- **BaselineAdminNetworkPolicy (BANP)**
+
+## General Notes
+
+- Any well defined AdminNetworkPolicy rules should
```
Comment (bpickard22) [inline site-src/guides/api-overview.md:30]: *rule

File: site-src/guides/api-overview.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+block traffic with malicious in-cluster clients, or workloads that pose security risks.
+Those traffic restrictions can only be lifted once the `Deny` rules are deleted,
+modified by the admin, or overridden by a higher priority rule.
+
+On the other hand, the `Allow` rules can be used to call out traffic in the cluster
+that needs to be allowed for certain components to work as expected (egress to
+CoreDNS for example). Those traffic should not be blocked when developers apply
+NetworkPolicy to their Namespaces which isolates the workloads.
+
+AdminNetworkPolicy `Pass` rules allows an admin to delegate security posture for
```
Comment (bpickard22) [inline site-src/guides/api-overview.md:72]: what does security posture mean here?

File: site-src/guides/api-overview.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+Integer priority values were added to the AdminNetworkPolicy API to allow Cluster 
+Admins to express direct and intentional ordering between various ANP Objects.  
+The `Priority` field in the ANP spec is defined as an integer value 
+within the range 0 to 1000 where rules with lower priority values have higher 
+precedence. Regardless of priority value all ANPs have higher precedence than 
+any defined NetworkPolicy or BaselineAdminNetworkPolicy objects.
+
+### AdminNetworkPolicy Rules 
+
+Each ANP should define at least one `Ingress` or `Egress` relevant in-cluster traffic flows.
```
Comment (bpickard22) [inline site-src/guides/api-overview.md:91]: *traffic flows, and not flows. and

File: site-src/guides/api-overview.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+Admins to express direct and intentional ordering between various ANP Objects.  
+The `Priority` field in the ANP spec is defined as an integer value 
+within the range 0 to 1000 where rules with lower priority values have higher 
+precedence. Regardless of priority value all ANPs have higher precedence than 
+any defined NetworkPolicy or BaselineAdminNetworkPolicy objects.
+
+### AdminNetworkPolicy Rules 
+
+Each ANP should define at least one `Ingress` or `Egress` relevant in-cluster traffic flows. 
+and the Actions that should take place for those flows. In each `gress` rule the user
```
Comment (bpickard22) [inline site-src/guides/api-overview.md:92]: 'egress'

File: site-src/guides/api-overview.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+The `Priority` field in the ANP spec is defined as an integer value 
+within the range 0 to 1000 where rules with lower priority values have higher 
+precedence. Regardless of priority value all ANPs have higher precedence than 
+any defined NetworkPolicy or BaselineAdminNetworkPolicy objects.
+
+### AdminNetworkPolicy Rules 
+
+Each ANP should define at least one `Ingress` or `Egress` relevant in-cluster traffic flows. 
+and the Actions that should take place for those flows. In each `gress` rule the user 
+should AT THE MINIMUM define an `Action`, and at least one `AdminNetworkPolicyPeer`.
```
Comment (bpickard22) [inline site-src/guides/api-overview.md:93]: I would add a brief description of an AdminNetworkPolicyPeer here, as this is the first time that it is mentioned

File: site-src/guides/examples.md [hunk 7]
```diff
@@ -0,0 +1,156 @@
+## Example AdminNetworkPolicy API Object Definitions 
+
+These examples will start with the object yaml defintions used to implement the 
+[core use cases](../index.md#adminnetworkpolicy-api-user-stories). Please feel 
+free to contribute more examples that may seem relevant to other users :-). 
+
+### Sample Spec for Story 1: Deny traffic at a cluster level
```
Comment (bpickard22) [inline site-src/guides/examples.md:7]: for all of these I think we should add brief descriptions of the ANP's at work, to go with the image

Comment (bpickard22) [reply site-src/guides/examples.md:7]: also we should add something to show the priority at work to show a BANP getting overridden or something similar to that

Comment (bpickard22) [reply site-src/guides/examples.md:7]: another option would be to have a better link between the cases and these examples

File: site-src/index.md [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+`v1alpha1` version of the api. More information on how the community ended up here 
+can be found in the [API KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/2091-admin-network-policy)
+and in the accompanying [KEP PR](https://github.com/kubernetes/enhancements/pull/2522)
+
+Future API developments should all start with a **well-defined** and **intentional** 
+user story.
+
+### Story 1: Deny traffic at a cluster level
+
+As a cluster admin, I want to apply non-overridable deny rules
```
Comment (bpickard22) [inline site-src/index.md:32]: we should show an example networkpolicy that would not override the ANP

File: site-src/guides/api-overview.md (hunk 2 above)
Comment (astoycos) [reply site-src/guides/api-overview.md:30]: I'm referring to the various and often multiple "rules" defined in an AdminNetworkPolicy Here

File: site-src/guides/api-overview.md (hunk 3 above)
Comment (astoycos) [reply site-src/guides/api-overview.md:72]: I guess you can think of this as terminal action (i.e allow or drop) It just allows the developer, with network policies) to decide what happens to the traffic

File: site-src/guides/api-overview.md (hunk 5 above)
Comment (astoycos) [reply site-src/guides/api-overview.md:92]: I meant in(OR)e/gress rule here :)

File: site-src/guides/api-overview.md (hunk 6 above)
Comment (astoycos) [reply site-src/guides/api-overview.md:93]: +1 I want to add a section on "selectors" in a follow up!

File: site-src/guides/examples.md (hunk 7 above)
Comment (astoycos) [reply site-src/guides/examples.md:7]: +1 lets tackle this in a follow up

**Pull request merged**